platform/ui/src/components/AllInOneMenu/__stories__/allInOneMenu.stories.mdx

```diff
@@ -0,0 +1,137 @@
+import AllInOneMenu from '../AllInOneMenu';
+import AllInOneMenuItemPanel from '../AllInOneMenuItemPanel';
+import AllInOneMenuSubMenu from '../AllInOneMenuSubMenu';
+import AllInOneMenuItem from '../AllInOneMenuItem';
+import AllInOneMenuDividerItem from '../AllInOneMenuDividerItem';
+import AllInOneMenuHeaderItem from '../AllInOneMenuHeaderItem';
+import InputRange from '../../InputRange/index.js';
+import SwitchButton from '../../SwitchButton/index.js';
+
+import { ArgsTable, Story, Canvas, Meta } from '@storybook/addon-docs';
+import { createComponentTemplate } from '../../../storybook/functions/create-component-story';
+
+export const argTypes = {
+  component: AllInOneMenu,
+  title: 'Components/AllInOneMenu',
+};
+
+<Meta
+  title="Components/AllInOneMenu"
+  component={AllInOneMenu}
+/>
+
+export const AllInOneMenuTemplate = args => (
+  <div className="w-80">
+    <AllInOneMenu {...args}>
+      <AllInOneMenuItemPanel label="Panel 1">
+        <AllInOneMenuItem
+          label="Item 1"
+          key="0"
+          onItemClick={() => console.info('Item 1 clicked.')}
+        ></AllInOneMenuItem>
+        <AllInOneMenuItem
+          label="Item 2"
+          alternateLabel="Alt item 2"
+          key="1"
+        ></AllInOneMenuItem>
+        <div
+          className="all-in-one-menu-item my-1 flex w-full justify-center"
+          style={{ height: '50px' }}
+        >
+          <span>Arbitrary item component:</span>
+          <InputRange
+            minValue={1}
+            maxValue={10}
+            value={5}
+            onChange={() => {}}
+          ></InputRange>
+        </div>
+        <AllInOneMenuDividerItem key="2"></AllInOneMenuDividerItem>
+        <AllInOneMenuSubMenu
+          key="3"
+          menuLabel="Level 2"
```

Comment:
where is this `menuLabel` used? i don't see it in the UI

Reply:
It is supposed to be used for the 'Back to' text  whenever the sub menu is actually displayed.. However, there appears to be a problem with Story Book picking up the Common.json localization file and I am not sure how to fix it. Perhaps you can give me a hand with that?

That said. I will rename this `backLabel` and maybe that will help.

Reply:
Actually, instead of the Menu and SubMenu components using `useTranslation('Commom')`, I will simply use the `backLabel` to be the "back" text so users can use whatever text they want. That will add flexibility and fix the Story Book problem.